src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,80 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
```

Comment:
It's better to give this interface a more clear name, e.g. `TodoListProps`

Reply:
Got it) Thanks)